tests/pr_test.go

```diff
@@ -395,3 +396,116 @@ func TestRunVPCPatternSchematics(t *testing.T) {
 	err := options.RunSchematicTest()
 	assert.NoError(t, err, "Schematic Test had unexpected error")
 }
+
+func TestRunSLZExample(t *testing.T) {
```

Comment:
This is not a good name for this test. This is a test of the vsi extension.

Also, we need to look at this test more closely in a deeper dive, as I'm worried about a few things with a test this complex, for instance that tear down of ALL resources is done properly in all scenarios, and that the test will correctly work with a PR initiated from a fork.